src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.2.cpp

```diff
@@ -55,7 +55,7 @@ void MultiDiracDeterminant::BuildDotProductsAndCalculateRatios_impl(int ref,
     const size_t n = *it2;
     //ratios[count]=(count!=ref)?sign[count]*det0*CalculateRatioFromMatrixElements(n,dotProducts,it2+1):det0;
     if (count != ref)
-      ratios[count] = sign[count] * det0 * CalculateRatioFromMatrixElements(n, dotProducts, it2 + 1);
+      ratios[count] = sign[count] * det0 * CalculateRatioFromMatrixElements(n, dotProducts, &(*(it2 + 1)) );
```

Comment:
Please remove all `&(*`

Reply:
@anbenali can you do it?  This one is to transform the iterator into a pointer. will need to change more thing for the offload :)